app/controllers/api/v2/courses_controller.rb

```diff
@@ -37,20 +43,7 @@ def publish
         if @course.publishable?
           @course.publish_sites
           @course.publish_enrichment(@current_user)
-
-          response = ManageCoursesAPIService::Request.sync_course_with_search_and_compare(
-            @current_user.email,
-            @provider.provider_code,
-            @course.course_code
-          )
-
-          if response
-            head :ok
```

Comment:
Should this not be returning `head :ok`? Or is that being handled elsewhere? I don't see where if so.